integration_tests/test.py

```diff
@@ -0,0 +1,18 @@
+from cgi import test
+
+
+def adding (x, y):
+    return x + y
+
+print(adding(2, 1))
+
+
+
+def test_adding():
+    assert adding(2, 2) == 4
+
```

Comment:
You don't need empty lines at the end of the file.

Reply:
ok